.github/workflows/infra_plan.yaml

```
@@ -133,6 +143,10 @@ jobs:
             const fs = require('fs');
             const output = fs.readFileSync('${{ steps.directory.outputs.dir }}/plan_output_multiline.txt', 'utf8');
             const status = '${{ steps.plan.outcome }}'
+
+            // Construct the artifacts URL
+            const artifactsUrl = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}/artifacts`;
```

It is possible to get from the outputs of the upload-artifact step
```suggestion
            const artifactsUrl = `${{ steps.artifact-upload-step.outputs.artifact-url }}`;
```